html/lesson5/tutorial.md

```diff
@@ -425,6 +425,61 @@ In our case, we only want to change the style of every odd row. Achieving that i
 
 > Try changing odd to even
 
+
+### Transitions
+
+Let's take some **property** in our code and change it, making it somehow different.  E.g. The `background` of the two links named **btn**, and  let's change it on hover.  This is a 'change'.  Now we can have this affect take place during a number of seconds, this is a **Transition**.  
```

Comment:
I feel like the wording in this paragraph could be improved, but it's not essential. How about something like this though?

```
Let's modify a **property** in our code - e.g. the button link `background` colour - so that it changes when you hover over the button. By default, the change will happen instantaneously. However we can animate the effect so that it changes slowly over time, using the **transition** property.
```